posts/2021-11-30-RedHat-Connect-London.adoc

```diff
@@ -0,0 +1,64 @@
+
+Although audiences have engaged well with virtual conferences over the past 18 months, it was great to attend an in-person event. For many, it was the first in a long time but it was a fantastic way to re-engage with the community and enable attendees to network with OSS and enterprise software developers from many different organisations.
+
+
+== Open Liberty sessions:
+Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations. We had a great turnout at both of these sessions, with interesting questions raised by the audience and many even staying to view our demos during their lunch break! 
+
+image::/img/blog/RedHatConnectSummitLondon-1.png[Photograph of Grace presenting first Open Liberty session,width=70%,align="center"]
+
+The first session focused on creating event-driven reactive microservices through the use of OSS tools like Kafka, MicroProfile and Open Liberty. The audience were provided with an understanding of what the app architecture could offer to their own applications and then they were guided through how they could start implementing it themselves. If you're interested in this yourself, you can try out our Open Liberty guide that walks you through this: link:https://openliberty.io/guides/microprofile-reactive-messaging.html['Creating reactive Java microservices'].
+
+image::/img/blog/RedHatConnectSummitLondon-2.png[Screenshot of presentation,width=70%,align="center"]
+
+The other session focused on creating true-to-production tests for microservices with OSS tools like Microshed testing, test containers and Open Liberty. It's evident that containers have revolutionised how we design and build our applications but with this move to containers, the parity between our development, testing and production environments has drifted even further apart. The aim of this session was to show developers how to effectively test our cloud-native applications. We also gave a demo to show how simple it is to create and run a MicroShed test in Maven using our link:https://openliberty.io/guides/microshed-testing.html['Testing a MicroProfile or Jakarta EE application'] Open Liberty guide.
```

Comment:
Microshed testing, test containers and Open Liberty
--->
https://microshed.org/microshed-testing/[Microshed testing], https://www.testcontainers.org/[Testcontainers], and Open Liberty

It's evident that containers have revolutionised how we design and build our applications but with this move to containers, the parity between our development, testing and production environments has drifted even further apart.
--->
Although moving to containers has revolutionised how we design and build our applications,  the parity between our development, testing and production environments has also drifted even further apart.